src/client/theme-default/components/VPSocialLink.vue

```diff
@@ -1,7 +1,21 @@
 <script lang="ts" setup>
 import type { DefaultTheme } from 'vitepress/theme'
 import { computed } from 'vue'
-import { icons } from '../support/socialIcons'
+// import { icons } from '../support/socialIcons'
+
+const icons = {
```

Comment:
move this to UnoCSS safelist and map the icon with `i-vitepress:` prefix: rn all social icons in the final css bundle